### PR TITLE
Fill out experiments section a bit

### PR DESCRIPTION
**Pull request by sunahsuh**
Still a WIP but adds a few lines about different experimentation platforms and links to other docs

---
This change is [Reviewable](https://reviewable.io/reviews/mozilla/firefox-data-docs/32)

### CODE REVIEW

File: tools/experiments.md [hunk 1]
```diff
@@ -5,3 +5,30 @@ The work is being tracked in
 [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1341812).
 
 # Guide to our Experimental Tools
+
+## Shield
+* Shield is an addon-based experimentation platform with fine-tuned enrollment criteria. The system add-on landed in FF 53.
+* For the moment, it sends back data in its own `shield` type ping, so there's lots of flexibility in data you can collect.
+* Uses the Normandy server to serve out study “recipes” (?)
+* Annotates the main ping in the environment/experiments block
+* The shield system is itself a system add-on, so rolling out changes to the entire system is out-of-band with release trains
```
Comment (harterrt) [inline tools/experiments.md]: What does 'out of band' mean?

File: tools/experiments.md [hunk 2]
```diff
@@ -5,3 +5,30 @@ The work is being tracked in
...
+* S&I team are product owners and shepherd the study development and release process along
```
Comment (harterrt) [inline tools/experiments.md]: Can you expand the acronym in "S&I team" and include a mailing list?

File: tools/experiments.md [hunk 3]
```diff
@@ -5,3 +5,30 @@ The work is being tracked in
...
+* Opt-out experiments should be available soon?
+* Further reading:
+* https://wiki.mozilla.org/Firefox/SHIELD
```
Comment (harterrt) [inline tools/experiments.md]: Please add an additional level of indentation for the additional reading.

**Comment by harterrt** [comment]
Hey, @sunahsuh, just making sure this didn't fall off your radar.

**Comment by harterrt** [comment]
@sunahsuh friendly ping.

**Comment by sunahsuh** [comment]
Would the next one have been less friendly? :P Sorry, and I need to make another pass already since Shield now has opt-out capabilities!

**Pull request merged**